src/createAsyncThunk.ts

```diff
@@ -339,6 +339,7 @@ If you want to use the AbortController to react to \`abort\` events, please cons
             options.condition &&
             options.condition(arg, { getState, extra }) === false
           ) {
+            abortedPromise.catch(() => {})
```

Comment:
I know this was initially my idea but I'm debating a bit if it would not be more future-proof, if we did something like
```diff

+    let started = false;
      function abort(reason?: string) {
+      if (started) {
        abortReason = reason
        abortController.abort()
+      }
      }

// ....


          }
+         started = true;
          dispatch(pending(requestId, arg)
```

Reply:
Agreed on this.